test/int/adapter.test.ts

```diff
@@ -12,10 +12,14 @@ import { ExtendedDebugClient } from 'vscode-chrome-debug-core-testsupport';
 import * as testSetup from './testSetup';
 import { HttpOrHttpsServer } from './types/server';
 import { isWindows } from './testSetup';
+import * as puppeteer from 'puppeteer';
+import { IAttachRequestArgs } from '../../../vscode-chrome-debug-core/lib/src/debugAdapterInterfaces';
+import { execSync } from 'child_process';
+import { expect } from 'chai';
 
 const DATA_ROOT = testSetup.DATA_ROOT;
 
-suite('Chrome Debug Adapter etc', function () {
+suite('Chrome Debug Adapter etc', () => {
```

Comment:
Mocha suggests using normal functions instead of arrow functions, so we can access the this pointer if needed: https://mochajs.org/#arrow-functions

Reply:
I think you can still do this with arrow functions: https://stackoverflow.com/questions/15971167/how-to-increase-timeout-for-a-single-test-case-in-mocha. I would vote for using arrow functions here.

Reply:
Yeah, the reason I changed this to an arrow function was mainly just because of our lint rules that will flag any regular `function` declarations, and were red-lining the entire test in the editor. The other option is just to add tslint exceptions for all the tests, but that's also kind of annoying. There's a big long discussion about arrow functions vs normal [here](https://github.com/mochajs/mocha/issues/2018). There's been some chatter about redesigns in mocha to better support arrow functions, but so far it doesn't seem to have gone anywhere.

Reply:
FWIW, Jasmine also recommends `function` over `()=>{}`
https://stackoverflow.com/questions/38915090/are-arrow-functions-recommended-in-jasmine-test-cases.

Personally, I'd prefer to use arrow functions almost everywhere since it makes javascript behave more like C#, Java, and others in regards to `this`, and of course I'm more familiar with those languages, but it seems like the JS community (at least right now) still relies pretty heavily on the current `this` behavior.

So I can go either way on this.

Reply:
Let's use the arrow function then. If we ever need to do something which the only way to do it is to get the this pointer, then we can change that particular call to use a function instead.